sdk/containerregistry/azure-containerregistry/azure/containerregistry/_container_registry_client.py

```diff
@@ -733,10 +733,9 @@ def update_repository_properties(self, *args, **kwargs):
         properties.can_list = kwargs.pop("can_list", properties.can_list)
         properties.can_read = kwargs.pop("can_read", properties.can_read)
         properties.can_write = kwargs.pop("can_write", properties.can_write)
-        properties.teleport_enabled = kwargs.pop("teleport_enabled", None)
```

Comment:
this seems like a breaking change, just want to make sure this change makes sense

Reply:
I have asked in group and got confirmed that it is expected to be removed.